python/ray/rllib/pg/policy.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+import ray
+from ray.rllib.models.catalog import ModelCatalog
+
+
+class PGPolicy():
+
+    other_output = []
+    is_recurrent = False
+
+    def __init__(self, registry, ob_space, ac_space, config):
+        name = "local"
+        self.registry = registry
+        self.local_steps = 0
+        self.config = config
+        worker_device = "/job:localhost/replica:0/task:0/cpu:0"
+        self.g = tf.Graph()
+        with self.g.as_default(), tf.device(worker_device):
```

Comment:
I'm not so sure about the use of `worker_device` here - I can take a look into this, and chances are we don't actually need it.

Reply:
Yeah you don't want to do this, TF will pick devices better by itself.

Reply:
You also shouldn't need the tf.Graph thing, agents already set up a default graph. In fact this might mess up checkpointing.

Reply:
yeah, this is copied from A3C so I should fix it there too